cli/githubutil.py

```diff
@@ -82,4 +86,47 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def remove_outdated_webhooks(
+    github_org_name: str,
+    github_cfg_name: str,
+    concourse_cfg_name: str,
+):
+    '''
+    Removes webhooks with query parameter 'concourse_id' whose value does not match
+    parameter 'concourse_cfg_name'. Other webhooks are not processed
+    '''
+    cfg_factory = ctx().cfg_factory()
+    github_cfg = cfg_factory.github(github_cfg_name)
+
+    github_api = _create_github_api_object(github_cfg=github_cfg)
+    github_org = github_api.organization(github_org_name)
+
+    for repository in github_org.repositories():
+        print("")
```

Comment:
do not use print directly; use util.info, util.debug, util.verbose, util.warning